### DISCUSSION TITLE
How create hidden folder?

### DISCUSSION THREAD

**Discussion by deamon2k** [rejetto/hfs#239] [Q&A]
How in HFS2. When the folder in the sheet is not visible, but you can go to the link.

**Comment by rejetto** [comment]
1. add the folder in "shared files", or click on the folder if you already added it
2. in the "who can see", click "no one"

<img width="669" alt="image" src="https://user-images.githubusercontent.com/1367199/236613739-b1c6c1b0-d732-41d5-9a60-8643e4ae0e05.png">

**Reply by deamon2k** [reply]
I did. But when you click on the link, there is nothing there. 
When you put "Anyone" and the link to the folder is visible and files.

**Comment by rejetto** [comment]
oh, you are right.
This is a bit hard to do.
I must absolutely make it easier, as it's a common need, but for the moment you need to add this in the "masks" field.

> "*": { can_see: true }

**Reply by deamon2k** [reply]
thx!

**Comment by rejetto** [comment]
while the "masks" option will stay, as "masks" is a tool that can do a lot of stuff,
i'm willing to introduce an option to easily cover this case

<img width="282" alt="image" src="https://user-images.githubusercontent.com/1367199/236634163-37a181b6-c732-44e1-baa5-af38052667bb.png">

You will be able to switch off this option to get what you asked, instead of configuring the "masks".
Do you think it's clear enough?
